whitesource.config

```diff
@@ -0,0 +1,375 @@
+resolveAllDependencies=false
+#excludeDependenciesFromNodes=.*commons-io.*,.*maven-model
+
+#npm.resolveDependencies=false
+#npm.ignoreSourceFiles=false
+#npm.includeDevDependencies=true
+#npm.runPreStep=true
+#npm.ignoreNpmLsErrors=true
+#npm.ignoreScripts=true
+#npm.yarnProject=true
+#npm.accessToken=
+#npm.identifyByNameAndVersion=true
+#npm.yarn.frozenLockfile=true
+#npm.resolveMainPackageJsonOnly=true
+#npm.removeDuplicateDependencies=false
+#npm.resolveAdditionalDependencies=true
+#npm.failOnNpmLsErrors = 
+#npm.projectNameFromDependencyFile = true
+#npm.resolveGlobalPackages=true							   
+#npm.resolveLockFile=true
+
+#bower.resolveDependencies=false
+#bower.ignoreSourceFiles=true
+#bower.runPreStep=true
+
+#nuget.resolvePackagesConfigFiles=false
+#nuget.resolveCsProjFiles=false
+#nuget.resolveDependencies=false
+#nuget.restoreDependencies=true
+#nuget.preferredEnvironment=
+#nuget.packagesDirectory=
+#nuget.ignoreSourceFiles=false
+#nuget.runPreStep=true
+#nuget.resolveNuspecFiles=false
+#nuget.resolveAssetsFiles=true
+
+#python.resolveDependencies=false
+#python.ignoreSourceFiles=false
+#python.ignorePipInstallErrors=true
+#python.installVirtualenv=true
+#python.resolveHierarchyTree=false
+#python.requirementsFileIncludes=requirements.txt
+#python.resolveSetupPyFiles=true
+#python.runPipenvPreStep=true
+#python.pipenvDevDependencies=true
+#python.IgnorePipenvInstallErrors=true
+#python.resolveGlobalPackages=true	
+#python.localPackagePathsToInstall=/path/to/local/dependency.egg, /path/to/local/dependency.zip	
+#python.resolvePipEditablePackages
+#python.path=/path/to/python
+#python.pipPath=/path/to/pip
+#python.runPoetryPreStep=true
+#python.includePoetryDevDependencies=true					  
+
+#maven.ignoredScopes=test provided
+#maven.resolveDependencies=false
+#maven.ignoreSourceFiles=true
+#maven.aggregateModules=true
+#maven.ignorePomModules=false
+#maven.runPreStep=true
+#maven.ignoreMvnTreeErrors=true
+#maven.environmentPath=
+#maven.m2RepositoryPath=
+#maven.downloadMissingDependencies=false
+#maven.additionalArguments=
+#maven.projectNameFromDependencyFile=true										 
+
+resolveAllDependencies=false
+archiveExtractionDepth=7
+followSymbolicLinks=true
+gradle.resolveDependencies=true
+gradle.aggregateModules=true
+gradle.preferredEnvironment=wrapper
+maven.resolveDependencies=true
+maven.runPreStep=true
+maven.aggregateModules=true
+maven.ignoredScopes=None
+html.resolveDependencies=true
+npm.resolveDependencies=true
+npm.runPreStep=true
+npm.yarnProject=true
+go.resolveDependencies=true
+go.collectDependenciesAtRuntime=true
+go.dependencyManager=
+python.resolveDependencies=true
+python.ignoreSourceFiles=true
+python.runPipenvPreStep=true
+python.pipenvDevDependencies=true
+python.requirementsFileIncludes=dev-requirements.txt
+python.installVirtualenv=true
+ruby.resolveDependencies=true
+ruby.ignoreSourceFiles=false
+ruby.installMissingGems=true
+ruby.runBundleInstall=true
+ruby.overwriteGemFile=true 
```

Comment:
Can we move this section to the top? It is hard to read inbetween the other config. Also where is the `dev-requirements.txt` from line 163

Reply:
It's in other python repos like this one. https://github.com/opensearch-project/opensearch-py/blob/main/dev-requirements.txt

Reply:
But this configuration is specific to our repository right?

Reply:
Yeah right. I am still using the same unified agent configuration for all since we want to see the performance of local config for WhiteSource integration. We are now only using the default one.

Reply:
+1 @gaiksaya  It's hard to read the file to find the uncommented items. Apart from moving the uncommented portion to the top, another question I would ask is, is it necessary to keep the commented items? Can we get rid of it?

Reply:
Was wondering if we should separate the actual config with the commented one or just remove it. Reason being if we need to add something or debug we can look at the commented config and see if anything applied to our use case. @zelinh you can decide. I am okay with anything, Thanks!

Reply:
We will always have the general configuration with all commented part here to track. https://github.com/opensearch-project/opensearch-build/blob/main/tools/vulnerability-scan/wss-unified-agent.config. Therefore we could have this one clean and customized for our repo.